src/Task/Attribute/CreateUpdateEntityTask.php

```diff
@@ -105,50 +116,60 @@ public function __invoke(PipelinePayloadInterface $payload): PipelinePayloadInte
             throw new ApiNotConfiguredException();
         }
 
-        if (!$payload->getResources() instanceof ResourceCursorInterface) {
-            throw new NoAttributeResourcesException('No resource found.');
-        }
-
         try {
             $excludesAttributes = $this->excludedAttributesProvider->getExcludedAttributes();
-
+            $isEnterprise = $apiConfiguration->isEnterprise() ?? false;
```

Comment:
Maybe you should use https://github.com/synolia/SyliusAkeneoPlugin/pull/73/files#diff-1cd56ef8a27450489cf1349bb626d3b3a1c4c8a32180450581cd7916a3d54754R79

Reply:
fixed in commit 3f521e9319e96e2aca784b1d4b43965bccca3fb5